experimental/rules_python/wheelmaker.py

```diff
@@ -157,12 +158,26 @@ def add_metadata(self, extra_headers, description, classifiers, requires,
         metadata += "\n"
         self.add_string(self.distinfo_path('METADATA'), metadata)
 
-    def add_entry_points(self, console_scripts):
+    def add_entry_points(self, console_scripts, plugins):
```

Comment:
I'd change it so that the .bzl file just merges console_scripts and plugins, and wheelmaker just always sees only entry_points.

Reply:
Done.